### ISSUE TITLE
 scrollbar to ng2-table

### ISSUE THREAD

**Issue by priyakokas** [valor-software/ng2-table#527]
can anyone tell me how to add scrollbar to ng2-table

**Comment by NABEEL-AHMED-JAMIL** [comment]
used the div for adding the scrollbar to ng2-table

**Comment by NABEEL-AHMED-JAMIL** [comment]
Add the scroll bar on the <-------------(css)--------------> 
<div class="row-fluid">
        <div (css)>
            <ng-table class="sortable" [config]="config" (tableChanged)="onChangeTable(config)" (cellClicked)="onCellClick($event)" [rows]="rows" [columns]="columns">

            </ng-table>
        </div>
        <pagination *ngIf="config.paging" class="pagination-sm" [(ngModel)]="page" [totalItems]="length" [itemsPerPage]="itemsPerPage" [maxSize]="maxSize" [boundaryLinks]="true" [rotate]="false" (pageChanged)="onChangeTable(config, $event)" (numPages)="numPages = $event">
        </pagination>
    </div>

**Comment by priyakokas** [comment]
thanks for your reply NABEEL but the code you gave i.e. of <Pagination> how that is going to help me to add vertical scroll bar for grid?

**Comment by NABEEL-AHMED-JAMIL** [comment]
// add this in css file
.ScrollStyle
{
    max-height: 150px;
    overflow-y: scroll;
}

**Comment by NABEEL-AHMED-JAMIL** [comment]
`<div class="ScrollStyle"><ng-table></ng-table></div>`

**Comment by NABEEL-AHMED-JAMIL** [comment]
i hope this work for you

**Comment by priyakokas** [comment]
Yes NABEEL it works for horizontal scroll. but in case of vertical scroll the table header is also moving. so I didnt applied it for vertical scroll.
